### PR TITLE
Drop more deprecated APIs

### PR DESCRIPTION
**Pull request by kgaillot**
... with a wee bit of best practices thrown in

### CODE REVIEW

**Comment by kgaillot** [comment]
@nrwahl2 , here's another one

**Review by nrwahl2 — COMMENTED** [review]
Nice

File: lib/pengine/unpack.c [hunk 1]
```diff
@@ -952,9 +952,7 @@ unpack_tags(xmlNode *xml_tags, pcmk_scheduler_t *scheduler)
                 continue;
             }
 
-            if (add_tag_ref(scheduler->tags, tag_id, obj_ref) == FALSE) {
-                return FALSE;
-            }
+            pcmk__add_idref(scheduler->tags, tag_id, obj_ref);
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply lib/pengine/unpack.c:943]: I'll leave that for a scheduler best practices PR

File: lib/pengine/pe_output.c [hunk 2]
```diff
@@ -3301,19 +3312,17 @@ ticket_xml(pcmk__output_t *out, va_list args) {
     bool raw G_GNUC_UNUSED = va_arg(args, int);
     bool details G_GNUC_UNUSED = va_arg(args, int);
 
-    const char *status = NULL;
-    const char *standby = pcmk__btoa(ticket->standby);
+    const char *standby = pcmk__btoa(pcmk_is_set(ticket->flags,
```
[the remark this answers is not in the document]
Comment (kgaillot) [reply lib/pengine/pe_output.c]: Ah right, thanks. Updated commit

**Review by nrwahl2 — APPROVED** [review]

**Pull request merged**